src/constants/tokenLists/mainnet.tokenlist.ts

```diff
@@ -245,19 +245,19 @@ export const MAINNET_TOKEN_LIST = {
     },
     {
       chainId: 1,
-      address: '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0',
-      symbol: 'wstETH',
-      name: 'Wrapped stETH (Lido)',
+      address: '0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84',
+      symbol: 'stETH',
+      name: 'Lido Staked ETH',
       decimals: 18,
-      logoURI: 'https://assets.coingecko.com/coins/images/18834/large/wstETH.png',
+      logoURI: 'https://s2.coinmarketcap.com/static/img/coins/64x64/8085.png',
     },
     {
       chainId: 1,
-      address: '0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84',
-      symbol: 'stETH',
-      name: 'stETH (Lido)',
+      address: '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0',
+      symbol: 'wstETH',
+      name: 'Lido wstETH',
       decimals: 18,
-      logoURI: 'https://assets.coingecko.com/coins/images/13442/large/steth_logo.png',
+      logoURI: 'https://lido.fi/static/images/favicon/apple-touch-icon.png',
```

Comment:
Why would you changing dis @viet-nv You are using lower quality img

Reply:
I added this on my branch before you add it, All information were got from cmc then when merging code from main branch, I accepted my code, not yours, haha

Reply:
:tien-sword: